packages/jam/transition/accumulate/accumulate.ts

```diff
@@ -532,6 +537,10 @@ export class Accumulate {
         return resultEntry;
       });
 
+      if (this.options.accumulateSequentially === true) {
+        await promise;
+      }
```

Comment:
here is the important change. the rest of the changes are needed to add the CLI option 🙈